pkg/config/config_test.go

```diff
@@ -63,9 +79,28 @@ metadata:
   name: my-kcl-fn
 data:
   param1: foo
+spec:
+  matchConstraints:
```

Comment:
Can we omit the empty property?

Reply:
Yeah, you're right, the `spec` should be empty in this case

Reply:
I mean that if I don't write the property what will be happen? Could you please add a test or example?